docs/portal.properties-Reference.md

```diff
@@ -215,6 +215,13 @@ The CIViC API url is set to https://civic.genome.wustl.edu/api/ by default. It c
 civic.url=
 ```
 
+# MDACC Heatmap Integration
+
+MDACC Heatmap integration (button in OncoPrint heatmap dropdown and tab on Study page can be turned on or off by setting the following property:
+```
+show.mdacc.heatmap=true
```

Comment:
If we don't have these documented yet:

```
            "mdacc.heatmap.meta.url",
            "mdacc.heatmap.patient.url",
            "mdacc.heatmap.study.meta.url",
            "mdacc.heatmap.study.url",
```

It might make sense to add those as well? B/c the mdacc heatmap won't work unless you have all these other properties right?

Reply:
@inodb @alisman Thank you both for all your help!

The first two are now unused, since they were for the patient view, and are only referenced in getMDAndersonHeatMapMetaUrl and getMDAndersonHeatmapPatientUrl, respectively.  Neither of these are called in the current frontend.  The patient view feature does work if we ever want to reinstate it.  i.e. The two properties could be left out without harm.

The next two are used by the study view tab, but they are effectively constants.
The new button in the OncoPrint does not depend on any of these, as it currently has hard-coded urls that will be changing (at our end) which will prompt minor code changes at some point.

Please let me know what you need from me.

Reply:
Thanks for clarifying @ChrisWakefield ! Then maybe ok to ignore those properties in the docs for now. Another option would be to have some section "internal use only". That way people browsing the docs can be made aware of the integration, but that it's not working yet / under development